src/objects/texts/zcl_abapgit_sotr_handler.clas.abap

```diff
@@ -21,9 +18,23 @@ CLASS zcl_abapgit_sotr_handler DEFINITION
       RAISING
         zcx_abapgit_exception .
     CLASS-METHODS create_sotr
+      IMPORTING
+        !iv_package  TYPE devclass
+        !io_xml      TYPE REF TO zif_abapgit_xml_input OPTIONAL
+        !it_sotr     TYPE zif_abapgit_definitions=>ty_sotr_tt OPTIONAL
+        !it_sotr_use TYPE zif_abapgit_definitions=>ty_sotr_use_tt OPTIONAL
```

Comment:
create one method for each option?

https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#prefer-multiple-static-creation-methods-to-optional-parameters

Reply:
@mbtools, see this comment